.github/workflows/pr_crictl.yml

```diff
@@ -48,27 +48,44 @@ jobs:
         rpm -ivh rune-${{ env.RUNE_VERSION }}-1.el8.x86_64.rpm;
         rpm -ivh shim-rune-${{ env.RUNE_VERSION }}-1.el8.x86_64.rpm'
 
-    - name: Run occlum-hello pod
-      if: always()
+    - name: Start containerd
       run: docker exec $rune_test bash -c "containerd" &
 
-        docker exec $rune_test bash -c "cd /root/samples && ./clean.sh;
-        crictl run --timeout 30s hello.yaml pod.yaml && ./show.sh"
+    - name: Run occlum-hello pod
+      uses: nick-invision/retry@v2
+      with:
+        timeout_minutes: 20
+        max_attempts: 5
+        command: docker exec $rune_test bash -c "cd /root/samples && ./clean.sh;
+          crictl run --timeout 90s hello.yaml pod.yaml && ./show.sh"
+        on_retry_command: echo "retry to run occlum-hello pod!!"
```

Comment:
Use a common pattern to highlight retry command: "******************[RETRY %d]******************"

Reply:
`nick-invision/retry@v2` does NOT impose the retry order for the user. If this step is retried, then `nick-invision/retry@v2`  will output the message: `Warning: Attempt 1 failed. Reason: Child_process exited with error code 1`, which is easy to find.